cog/command.py

```diff
@@ -66,15 +61,15 @@ def options(self):
         return dict(self.options_)
 
     def arg_count(self):
-        return self.argc_
+        return len(self.args_)
 
     def arg(self, index):
         if index >= self.argc_:
-            return nil
+            return ""
```

Comment:
Thanks for spotting my error!

I've thought about returning `None` vs. `""` here and I still believe asking for an out of bounds index shouldn't return a potentially valid value.

I'd be receptive to a PR which returns either `None` or raises `IndexError` here. If we decide to raise then we should also raise in `option()`. `KeyError` seems like it'd be appropriate.